polkadot-protocol/architecture/polkadot-chain/overview.md

```diff
@@ -1,4 +1,260 @@
+description: Explore the fundamental concepts of the Polkadot protocol, including account addresses, extrinsics, events and runtime upgrades.
+---
+
+# Overview
+
+Since the release of Bitcoin in 2009, blockchain projects increased exponentially to the order of tens of thousands. 
+
+Different projects have different value propositions, suggesting that the future will be multi-chain and that inter-chain communication will be crucial to establish collaborations and leveraging each other strengths.
+
+## Polkadot 1.0
+
+Polkadot 1.0 represents the state of Polkadot as of 2023, coinciding with the release of [Polkadot runtime v1.0.0](https://github.com/paritytech/polkadot/releases/tag/v1.0.0){target=\_blank}. This section will focus on Polkadot 1.0, along with philosophical insights into network resilience and blockspace.
+
+As a Layer-0 blockchain, Polkadot contributes to the multi-chain vision through several key innovations and initiatives, including:
+
+1. **Application-specific Layer-1 blockchains (parachains)** - Polkadot's sharded network allows for parallel transaction processing, with shards that can have unique state transition functions, enabling custom-built L1 chains optimized for specific applications
+
+2. **Shared security and scalability** - L1 chains connected to Polkadot benefit from its [Nominated-Proof-of-Stake (NPoS)](link to the PoS page) system, providing security out-of-the-box without the need to bootstrap their own
+
+3. **Secure interoperability** - Polkadot's native interoperability enables L1 and L2 chains to seamlessly exchange data and value across parachains
+
+4. **Resilient infrastructure** - decentralized and scalable, Polkadot ensures ongoing support for development and community initiatives via its on-chain [treasury](https://wiki.polkadot.network/docs/learn-polkadot-opengov-treasury){target=\_blank} and governance
+
+5. **Rapid L1 development** - the [Polkadot SDK](link to Polkadot SDK page in polkadot protocol section) allows fast, flexible creation and deployment of Layer-1 chains
+
+6. **Cultivating the next generation of Web3 developers** - Polkadot supports the growth of Web3 core developers through initiatives such as:
+
+    - [Polkadot Blockchain Academy](https://polkadot.com/blockchain-academy){target=\_blank}
+    - [Polkadot Alpha Program](https://polkadot.com/alpha-program){target=\_blank}
+    - [Polkadot Developer Heroes Program](https://wiki.polkadot.network/docs/dev-heroes){target=\_blank}
+    - [EdX Courses](https://www.edx.org/school/web3x){target=\_blank}
+    - Rust and Substrate Courses (coming soon)
+
+### High Level Architecture
+
+Polkadot features a chain that serves as the central component of the system. This chain is depicted as a ring, encircled by several parachains that are connected to it. 
+
+According to Polkadot's design, any chain that can compile to WebAssembly (Wasm) and complies with the  chain's API can integrate into the Polkadot network as a parachain.
```

Comment:
Parachains/ELVES protocol should be linked here. It could also be useful to mention Substrate/Polkadot SDK as the means most people use to build these chains.

```suggestion
According to Polkadot's design, any blockchain that can compile to WebAssembly (Wasm) and adheres to the Parachains Protocol become a parachain on the Polkadot network.
```